.github/CODEOWNERS

```diff
@@ -206,6 +206,47 @@
 /tactics/class_tactics.* @coq/typeclasses-maintainers
 /test-suite/typeclasses/ @coq/typeclasses-maintainers
 
+########## Number ##########
+
+/interp/numTok.*                     @coq/number-maintainers
+/kernel/float64*                     @coq/number-maintainers
+/kernel/uint63*                      @coq/number-maintainers
+/plugins/syntax/g_number_string.mlg  @coq/number-maintainers
+/plugins/syntax/int63_syntax_plugin.mllib  @coq/number-maintainers
+/plugins/syntax/number.ml            @coq/number-maintainers
+/plugins/syntax/number_string_notation_plugin.mllib  @coq/number-maintainers
+/user-contrib/Ltac2/Int.v            @coq/number-maintainers
+/test-suite/output/FloatExtraction.v  @coq/number-maintainers
+/test-suite/output/FloatSyntax.v     @coq/number-maintainers
+/test-suite/output/Int31Syntax.v     @coq/number-maintainers
+/test-suite/output/Int63Syntax.v     @coq/number-maintainers
+/test-suite/output/NSyntax.v         @coq/number-maintainers
+/test-suite/output/NumberNotations.v  @coq/number-maintainers
+/test-suite/output/RealSyntax.v      @coq/number-maintainers
+/test-suite/output/sint63Notation.v  @coq/number-maintainers
+/test-suite/output/Sint63Syntax.v    @coq/number-maintainers
+/test-suite/output/ZSyntax.v         @coq/number-maintainers
+/test-suite/primitive/float/         @coq/number-maintainers
+/test-suite/primitive/sint63/        @coq/number-maintainers
+/test-suite/primitive/uint63/        @coq/number-maintainers
+/theories/Init/Decimal.v             @coq/number-maintainers
+/theories/Init/Hexadecimal.v         @coq/number-maintainers
+/theories/Init/Nat.v                 @coq/number-maintainers
+/theories/Init/Number.v              @coq/number-maintainers
+/theories/Arith/                     @coq/number-maintainers
```

Comment:
That's a big chunk. If that's included, then @MSoegtropIMC might be interested in being in this team.